kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -36,7 +36,35 @@ const fetchNodes = ({ dispatch }, id) => {
   });
 };
 
+
+const searchNodes = ({ dispatch }, params, page) => {
+  // Get the collection from ContentNodeResource.
+  const contentCollection = Kolibri.resources.ContentNodeResource.getCollection();
+  const wantedFields = ['pk', 'title', 'kind', 'instance_id', 'content_id', 'description', 'files'];
```

Comment:
Here's an example:
![image](https://cloud.githubusercontent.com/assets/1680573/16671798/7762adaa-4456-11e6-85a3-107392ce057b.png)
We save about 4Kb per request by doing the field filtering - not a huge result, especially as it means we have to do a second request to get the full field data afterwards.